datasette/templates/__sorting_example.html

```diff
@@ -0,0 +1,29 @@
+<script src="{{ urls.static('sort_utils.js') }}"></script>
+<div>
+    <p style="display:inline-block;">Sorting options:</p>
+    <button type="button" class="btn-secondary" id="toggle_sort_menu" onclick="toggleSortMenu();" style="display:inline-block;">show</button>
+</div>
+<div style="display:block;position: absolute;z-index: 1;">
```

Comment:
nit: add spaces after semicolons and colons
```suggestion
<div style="display: block; position: absolute; z-index: 1;">
```